src/AzurePipelineParser/Models/YamlStreamProviders/IYamlStreamProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+using System.IO;
+
+namespace AzurePipelineParser.Models.YamlStreamProviders
+{
+    public interface IYamlStreamProvider
+    {
+        Stream GetStreamFromPath(string path); 
```

Comment:
NIT: should this be async?
```suggestion
        Task<Stream> GetStreamFromPathAsync(string path); 
```